packages/ui/react-ui-data/src/translations.ts

```diff
@@ -16,6 +16,12 @@ export default [
         'delete field': 'Delete field',
         'field limit reached': 'Maximum number of fields reached',
 
+        'button add property': 'Add property',
+
+        // TODO(burdon): Factor out?
```

Comment:
A lot of common translations are in the `'os'` namespace defined in the shell, which has `'cancel label'` but currently no ‘save’.